### PR TITLE
Merge with terraform-dcos-enterprise

### PR DESCRIPTION
**Pull request by cprovencher**

### CODE REVIEW

File: azure/main.tf [hunk 1]
```diff
@@ -99,7 +99,7 @@ resource "azurerm_network_security_rule" "public-subnet-httpRule" {
     direction                   = "Inbound"
     access                      = "Allow"
     protocol                    = "Tcp"
-    source_port_range           = "80"
+    source_port_range           = "*"
```
Comment (cprovencher) [inline azure/main.tf]: @bernadinm  please advice here

File: azure/main.tf [hunk 2]
```diff
@@ -114,7 +114,7 @@ resource "azurerm_network_security_rule" "public-subnet-httpsRule" {
     direction                   = "Inbound"
     access                      = "Allow"
     protocol                    = "Tcp"
-    source_port_range           = "443"
+    source_port_range           = "*"
```
Comment (cprovencher) [inline azure/main.tf]: @bernadinm  please advice here

File: aws/bootstrap.tf [hunk 3]
```diff
@@ -242,6 +238,20 @@ resource "null_resource" "bootstrap" {
     agent = "${local.agent}"
   }
 
+  }
```
Comment (sesh-kebab) [inline aws/bootstrap.tf]: To run this locally, I've need to delete this extra brace.

Comment (sesh-kebab) [inline aws/bootstrap.tf]: This additional brace needs to be deleted.

**Review by sesh-kebab — CHANGES_REQUESTED** [review]
I've run this locally and can confirm that I've been able to bring up an EE dc/os cluster. Just the extra brace in `bootstrap.tf` needs to be removed.

File: aws/bootstrap.tf (hunk 3 above)
Comment (cprovencher) [reply aws/bootstrap.tf]: @sesh-kebab  thanks!

**Comment by cprovencher** [comment]
aws/docs/open-to-enterprise_module_only.md
needs to be modified or removed

**Pull request closed without merging**